system/jlib/jptree.cpp

```diff
@@ -7630,27 +7733,15 @@ void mergeConfiguration(IPropertyTree & target, IPropertyTree & source)
         //Legacy support for old roxie configuration files that have repeated elements with no name tag
         if (!name)
             name = child.queryProp("@netAddress");
```

Comment:
Is this still necessary ?
Not particularly nice being hard-wired into jptree, could you instead pass it in, and use only in the roxie loadConfiguration context?